cmd/agentctl/commands/config.go

```diff
@@ -108,6 +111,9 @@ func runConfigGet(cli agentcli.Cli, opts ConfigGetOptions) error {
 	if err != nil {
 		return fmt.Errorf("parsing labels failed: %w", err)
 	}
+	if len(opts.Labels) == 0 && !opts.All {
+		labels["io.ligato.from-client"] = "agentctl"
```

Comment:
I do not think this is a good idea for a default filtering. For the update command it's definitely good to add that info. However in most scenarios agentctl works on top of other configuration and this behavior might confuse users if they see empty configuration when running `agentctl config get` first time.

Reply:
Done.